### PR TITLE
transfer database to new project

### PR DESCRIPTION
**Pull request by Stefann11**

### CODE REVIEW

File: PatientWebApplication/HealthClinic.DAL/Dtos/FeedbackDto.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+{
+    public class FeedbackDto
+    {
+        public string Message { get; set; }
+        public bool IsPublic { get; set; }
+        public bool IsAnonymous { get; set; }
+        public int PatientId { get; set; }
+
+        public FeedbackDto() { }
+    }
```
Comment (Nemanja0505) [inline PatientWebApplication/HealthClinic.DAL/Dtos/FeedbackDto.cs:16]: Add constructor with parameters

File: PatientWebApplication/HealthClinic.BL/Adapters/FeedbackAdapter.cs [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
-using PatientWebApplication.Dtos;
+﻿using HealthClinic.BL.Dtos;
+using HealthClinic.BL.Model.Patient;
 using System;
 
-namespace PatientWebApplication.Adapters
+namespace HealthClinic.BL.Adapters
 {
     public class FeedbackAdapter
     {
```
Comment (Nemanja0505) [inline PatientWebApplication/HealthClinic.BL/Adapters/FeedbackAdapter.cs:8]: Use constructor with parameters for new feedback in method FeedbackDtoToFeedback

Comment (Nemanja0505) [inline PatientWebApplication/HealthClinic.BL/Adapters/FeedbackAdapter.cs:8]: Use constructor with parameters for new FeedbackDto in method FeedbackToFeedbackDto

**Review by Nemanja0505 — CHANGES_REQUESTED** [review]
Pull request is not approved, fix the marked parts

**Review by Maja0505 — APPROVED** [review]

**Review by Nemanja0505 — APPROVED** [review]

**Pull request merged**